Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -0,0 +1,74 @@
+#include "pch.h"
+#include "EngineView.h"
+#include "EngineView.g.cpp"
+
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Xaml::Input;
+using namespace winrt::Windows::UI::Xaml::Media;
+using namespace winrt::Windows::UI::Xaml::Controls;
+
+namespace winrt::BabylonReactNative::implementation {
+    EngineView::EngineView() {
+        _revokerData.SizeChangedRevoker = SizeChanged(winrt::auto_revoke, { this, &EngineView::OnSizeChanged });
+        _revokerData.PointerPressedRevoker = PointerPressed(winrt::auto_revoke, { this, &EngineView::OnPointerPressed });

```

Comment:
For pointer events, rather than subscribing to the evented form, you should instead override the `OnPointer`* methods. These are invoked before the evented forms, and provide an option for suppressing the event from being raised (by setting the `Handled` property) in cases where that might not be valuable (think not raising `PointerPressed` for a button but instead a `Click`).

Reply:
sounds good, I'll add this to the upcoming packaging review.